### ISSUE TITLE
Support for Biomes o' Plenty.

### ISSUE THREAD

**Issue by SwordShaman** [An-Sar/PrimalCore#47]

**Comment by An-Sar** [comment]
Yes. Most likely in the 1.11+ branches, which is where development is now focused.

What version of minecraft are you using?

**Comment by SwordShaman** [comment]
I'm using 1.11.2 version.  Thanks for the quick reply, neighbor.  Keep up the great work!
